src/V3EmitCSyms.cpp

```diff
@@ -507,15 +517,15 @@ void EmitCSyms::emitSymHdr() {
         puts("\n// COVERAGE\n");
         puts(v3Global.opt.threads() > 1 ? "std::atomic<uint32_t>" : "uint32_t");
         puts(" __Vcoverage[");
-        puts(cvtToStr(m_coverBins));
+        puts(std::to_string(m_coverBins));
         puts("];\n");
     }
 
     if (!m_scopeNames.empty()) {  // Scope names
         puts("\n// SCOPE NAMES\n");
-        for (const auto& itr : m_scopeNames) {
-            putns(itr.second.m_nodep,
-                  "VerilatedScope " + protect("__Vscope_" + itr.second.m_symName) + ";\n");
+        for (const auto& pair : m_scopeNames) {
```

Comment:
I dislike `pair` as it looks like a datatype alias of `std::pair`, and does describes the data type not what it contains which is what it should.  Keep `itr` to match elsewhere (as it is containing an iteration reference), or `itpair`, or describe what is contained e.g. `nameData`.

Reply:
I'm looking forward to C++17 where all this would be just `for (const auto& [_, sd] : map)`.

Reply:
Just FWIW I dislike `itr` because it suggest it's an `std::map<K,V>::iterator` type, while it is a `std::map<K,V>::value_type`, which is defined by the standard to be `std::pair<const K, V>`, but I hear you regarding `using std::pair`. I changed them to `itpair`.